kafka-perf/commands.sh

```diff
@@ -25,10 +25,19 @@ fi
 
 
 # Create topics
-kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test-rep-one --partitions 6 --replication-factor 1
-kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test --partitions 6 --replication-factor 3 
+# kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test-rep-one --partitions 6 --replication-factor 1
+# kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test --partitions 6 --replication-factor 3 
```

Comment:
Why are these commented out? Can we remove or move them to a different file?

Reply:
Removed